added homepage

tusharkandpal opened the pull request:
- main page html based on figma design
- used wings-ui library & few own styles to override


![Recording 2022-02-12 at 16 45 27](https://user-images.githubusercontent.com/66052835/153709089-ba8dd21a-d4ea-430a-9289-a389be82621f.gif)

tusharkandpal committed:
added homepage

- main page html based on figma design
- used wings-ui library & few own styles to override

fahd23 reviewed (approved):
Looks good to me, it would be nice if you have a preview

tusharkandpal commented:
> Looks good to me, it would be nice if you have a preview

ya forgot it...adding now

shaksham08 commented on index.css:
```diff
@@ -0,0 +1,54 @@
+/* --------------------------- || CATEGORY SECTION ----------------- */
+.main {
+    padding: 2rem 7rem;
+    background-color: #F7F7F7;
```
you can use color variables instead of hardcoded colors

tusharkandpal replied:
i will create color variables at last.

shaksham08 reviewed (approved):
Looks good to me.
Just added one comment

tusharkandpal committed:
navigation changes

- imported wing library in main css
- created utility class for link
- added comments in homepage html
